src/components/ListItem.jsx

```diff
@@ -1,5 +1,61 @@
 import './ListItem.css';
+import { updateItem } from '../api/firebase';
+import { useState, useEffect } from 'react';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ id, name, listPath }) {
+	const [itemChecked, setItemChecked] = useState(false);
+
+	useEffect(() => {
+		const lastCheckedTime = localStorage.getItem(`checked_${id}`);
```

Comment:
Great usage of LocalStorage here! However, take in mind that the list is shared so, if several people is using the list, part of the functionality will only work for the user setting the checkbox, as Localstorage is not shared across users. You need to include lastCheckedTime in the Firebase document